src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -293,6 +293,10 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
     validityIntervalStart && validityIntervalStart.toString()
   )
 
+  const prepareMetaDataHashHex = (metaDataHash: Buffer | null): string | null => (
```

Comment:
Also in another PR, I would create `_Metadata` type, also `_Ttl` and `_ValidityIntervalStart` types.